text/009-rfc-process-addendum.md

```diff
@@ -0,0 +1,142 @@
+* 2019-01-12: Y signs off on the FCP proposal.
+* 2019-01-17: Z signs off on the FCP proposal. All team members have now signed
+  off, and the RFC enters FCP.
+* 2019-01-24: The seven-day FCP period has passed without additional concerns
+  being raised, so the RFC is merged.
+
+### X Does Not Respond to the FCP Proposal
+
+* 2019-01-01: The RFC is written and proposed. Community discussion begins.
+* 2019-01-10: Consensus has formed, and Z proposes that the RFC enter its FCP
+  with disposition to merge.
+* 2019-01-12: Y signs off on the FCP proposal.
+* 2019-01-24: X has not responded to the FCP in fourteen days, so they
+  automatically defer to Y and Z. The RFC enters its FCP.
+* 2019-01-31: The seven-day FCP period has passed without additional concerns
+  being raised, so the RFC is merged.
+
+### Y Requests a Time-to-Respond Extension and then Later Responds
+
+* 2019-01-01: The RFC is written and proposed. Community discussion begins.
+* 2019-01-10: Consensus has formed, and X proposes that the RFC enter its FCP
+  with disposition to merge.
+* 2019-01-12: Z signs off on the FCP proposal.
+* 2019-01-15: Y is going on vacation and hasn't had a chance to give the RFC
+  deep consideration. They request a time-to-respond extension.
+* 2019-01-28: Y comes back from vacation, considers the RFC, and signs off on
+  it. All team members have now signed off and the RFC enters FCP.
+* 2019-02-05: The seven-day FCP period has passed without additional concerns
+  being raised, so the RFC is merged.
+
+### Z Requests a Time-to-Respond Extension and then Fails to Respond
+
+* 2019-01-01: The RFC is written and proposed. Community discussion begins.
+* 2019-01-10: Consensus has formed, and X proposes that the RFC enter its FCP
+  with disposition to merge.
+* 2019-01-12: Y signs off on the FCP proposal.
+* 2019-01-13: Z requests a time-to-respond extension.
+* 2019-02-11: It has been 28 days since the date of the FCP proposal, and Z has
+  still not responded. Therefore, they automatically defer to X and Y. The RFC
+  enters its FCP.
+* 2019-02-18: The seven-day FCP period has passed without additional concerns
+  being raised, so the RFC is merged.
+
+# Drawbacks, Rationale, and Alternatives
+[alternatives]: #drawbacks-rationale-and-alternatives
+
+<!-- This is your chance to discuss your proposal in the context of the whole design -->
+<!-- space. This is probably the most important section! -->
+<!-- - Why is this design the best in the space of possible designs? -->
+<!-- - What other designs have been considered and what is the rationale for not -->
+<!--   choosing them? -->
+<!-- - What is the impact of not doing this? -->
+
+The primary drawback to this RFC is that it enables going forward with RFCs that
+have not been considered by every team member. The alternative, and our current
+state of affairs, is delaying the RFC until every team member has signed off on
+the RFC regardless how long that might take. There is a fundamental trade off
```

Comment:
I'm in favor of moving to FCP if we can remove the two final sentences of this paragraph.

Reply:
Done!